.npmrc

```diff
@@ -0,0 +1,2 @@
+auto-install-peers=false
+lockfile-exclude-links=false
```

Comment:
this was bugging me with pnpm installs - adding it so others avoid the same struggle